src/discord_mcp/core/plugins/manager.py

```diff
@@ -5,7 +5,15 @@
 
 from mcp.types import ToolAnnotations
 
-from discord_mcp.core.server.common.manifests import BaseManifest, PromptManifest, ResourceManifest, ToolManifest
+from discord_mcp.core.server.shared.manifests import BaseManifest, PromptManifest, ResourceManifest, ToolManifest
+
+__all__: tuple[str, ...] = (
+    "DiscordMCPPluginManager",
+    "ToolManifest",
+    "ResourceManifest",
+    "PromptManifest",
+)
```

Comment:
```suggestion
__all__: tuple[str, ...] = (
    "DiscordMCPPluginManager",
)
```
originally manifests were defined in the same file as the plugin manager, noone catched this from my autocomplete PR lmfao